scripts/rf_accounts.py

```diff
@@ -41,9 +50,41 @@
     logger = redfish.redfish_logger( log_file, log_format, logging.DEBUG )
     logger.info( "rf_accounts Trace" )
 
+user_uri = None
 # Set up the Redfish object
-redfish_obj = redfish.redfish_client( base_url = args.rhost, username = args.user, password = args.password )
-redfish_obj.login( auth = "session" )
+try:
+    redfish_obj = redfish.redfish_client( base_url = args.rhost, username = args.user, password = args.password , timeout=5, max_retry=3)
+    redfish_obj.login( auth = "session" )
+except RedfishPasswordChangeRequiredError as e:
+    if args.setpassword is None:
+        print_password_change_required_and_logout(redfish_obj, args)
+        sys.exit(1)
+    else:
+        user_uri = e.args[1]
+        if args.setpassword[0] == args.user:
+            pass
```

Comment:
Instead of doing a "pass" here, we should perform the password change request here like this:

```
redfish_utilities.set_password_via_uri( redfish_obj, <uri from exception>, args.setpassword[1] )
```

We can't reasonably follow the normal flow of the script because the traversal logic to look up the desired user account will not always resolve the ManagerAccount resource to PATCH. So, adding in a method to change an account password based on a URI from the exception will alleviate any potential traversal logic issues.